tools/topology/m4/utils.m4

```diff
@@ -101,33 +101,21 @@ define(`W_DATA',
 `}')
 
 dnl VIRTUAL_DAPM_ROUTE_OUT(name, dai type, dai index, direction, index)
+dnl No routes needed. Macro retained for simplicity
```

Comment:
Why not remove the macro then if it's not useful?

Reply:
i'd have to modify 4-5 tplg files to remove the macro. this way the change is one place and anyway we'll transition to 2.0 soon. so this wont be an issue